MPMap.html

```diff
@@ -0,0 +1,241 @@
+            aidRating.push("A" + k, "C" + k);
+        }
+
+        // Layers for markers and tooltips of climbs 
+        let rockLayer = L.layerGroup();
+        let iceLayer = L.layerGroup();
+        let mixedLayer = L.layerGroup();
+        let aidLayer = L.layerGroup();
+
+        // Arrays to hold grades of climbs
+        let rockDifficulty = [];
+        let iceDifficulty = [];
+        let mixedDifficulty = [];
+        let aidDifficulty = [];
+
+        // Temporary string to operate on if statement
+        let iceTemp;
+
+        // Set initial map view position
+        let map = L.map('map').setView([49.545206,-122.629395], 9);
+
+        // Terrain map layer
+        let OpenTopoMap = L.tileLayer('https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png', {
+            maxZoom: 17,
+            attribution: 'Map data: &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, <a href="http://viewfinderpanoramas.org">SRTM</a> | Map style: &copy; <a href="https://opentopomap.org">OpenTopoMap</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a>)'
+        });
+        OpenTopoMap.addTo(map);
+
+        //CORS header for querying, get access by following https://cors-anywhere.herokuapp.com/corsdemo
+        jQuery.ajaxPrefilter(function(options) {
+            if (options.crossDomain && jQuery.support.cors) {
+                options.url = 'https://cors-anywhere.herokuapp.com/' + options.url;
+            }
+        });
+        function getProfile() {
+            //Take the url user inputs to scrape
+            let userTodo = document.getElementById("mpUser").value;
+            $(document).ready(function() { 
+                $.get(userTodo, (html) => { 
+                    for(let i = 0; i < $(html).find("tr.route-row").length / 2; i++){
+
+                            // Find climbs on to do list
+                            currentBlock = $(html).find("tr.route-row:eq(" + i + ")").html();
+
+                            // String operations to find grades of climbs
+                            rockGrade = currentBlock.slice(currentBlock.indexOf("rateYDS") + 9);
+                            iceTemp = rockGrade;
+                            rockGrade = rockGrade.substring(0, rockGrade.indexOf("<")).trim();
+                            mixedGrade = currentBlock.slice(currentBlock.indexOf("British"));
+                            mixedGrade = mixedGrade.substring(mixedGrade.indexOf("</span>") + 7, mixedGrade.indexOf("<div>")).trim();
+                            
+                            // Last two characters in mixed grade will be A/C ratings, treat it as an aid climb if it is in aidRating (A/C 0-5)
+                            if(aidRating.includes(mixedGrade.slice(-2))){
+                                aidGrade = rockGrade + " " + mixedGrade;
+                                aidDifficulty.push(aidGrade);
+                                mixedDifficulty.push(null)
```

Comment:
What's the reason for pushing null here? is that to ensure all the arrays have the same number of indexes?